.forestry/front_matter/templates/fundraiser.yml

```diff
@@ -10,6 +10,17 @@ fields:
     min: 2
   label: Title
   description: The title of the fundraiser.
+- name: target
```

Comment:
*comment:* I see targets are back. I think this is contra-productive providing targets without the explanation where this target comes from. None of the fundraisers have this explained in their texts. I also think this is not helpful for open-ended projects, which all of them are. In practice they can have unlimited funding needs. So if this is not tied to clear milestones I advise against providing targets.

Reply:
Yes, I think milestones are really the best way to document needs. This could be modeled similar to the `allocations`, so a project can have multiple of those.